microservices/cmslitemetadata_to_redshift/Pipfile

```diff
@@ -12,6 +12,7 @@ botocore = "*"
 psycopg2-binary = "*"
 microservices = {editable = true,path = "./.."}
 numpy = "*"
+flake8 = "*"
```

Comment:
I will move this to the `[dev-packages]` section of the pipfile, since our linting tools are not required to run the application, but are rather only needed for development work.

```
[dev-packages]
flake8 = "*"
pylint = "*"
python-language-server = {extras = ["all"],version = "*"}
```

By using that section of the Pipfile, we can install the production dependencies independently of the development dependencies:

```
pipenv install --dev
```

see: https://pipenv-fork.readthedocs.io/en/latest/basics.html#pipenv-install for documentatin of this useage of Pipenv.

Reply:
I will also change the other Pipfiles affected in this PR accordingly.

Reply:
I will also update all `Pipfile.lock` files. Details on that in another comment: https://github.com/bcgov/GDX-Analytics/pull/288#issuecomment-721293849